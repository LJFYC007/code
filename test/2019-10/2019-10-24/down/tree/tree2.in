100 100
310 -840 244 -934 -333 248 -80 -534 -961 -712 720 -792 796 -165 -378 -720 888 385 -685 -32 275 -876 516 80 792 -516 -400 892 -248 308 780 -700 981 388 -630 -520 665 70 -920 8 -127 944 783 -520 -799 392 436 172 -584 493 2 622 -129 803 818 -495 -550 164 -958 456 -496 685 910 450 -290 -276 413 800 76 176 -93 178 -649 800 -384 576 922 -120 -700 242 -600 940 680 -700 984 782 -77 -400 -836 -212 -80 -736 -856 -848 725 724 598 995 114 -740 
1 2 1
2 3 1
3 4 3
4 5 2
5 6 3
6 7 3
7 8 3
8 9 2
9 10 2
10 11 2
11 12 1
12 13 3
13 14 2
14 15 2
15 16 3
16 17 1
17 18 3
18 19 2
19 20 2
20 21 3
21 22 2
22 23 2
23 24 2
24 25 2
25 26 2
26 27 1
27 28 1
28 29 1
29 30 3
30 31 2
31 32 2
32 33 1
33 34 3
34 35 1
35 36 3
36 37 2
37 38 3
38 39 2
39 40 3
40 41 2
41 42 2
42 43 1
43 44 2
44 45 3
45 46 3
46 47 3
47 48 1
48 49 3
49 50 1
50 51 2
51 52 3
52 53 3
53 54 1
54 55 2
55 56 1
56 57 3
57 58 3
58 59 3
59 60 1
60 61 3
61 62 2
62 63 2
63 64 3
64 65 1
65 66 3
66 67 1
67 68 2
68 69 1
69 70 2
70 71 2
71 72 3
72 73 1
73 74 3
74 75 2
75 76 1
76 77 3
77 78 1
78 79 2
79 80 2
80 81 2
81 82 3
82 83 2
83 84 2
84 85 2
85 86 3
86 87 3
87 88 2
88 89 2
89 90 3
90 91 2
91 92 1
92 93 1
93 94 1
94 95 2
95 96 1
96 97 3
97 98 2
98 99 2
99 100 1
1 37
8 18
23 81
18 3
10 14
44 58
26 24
60 25
89 92
4 58
13 68
56 49
58 34
43 45
64 81
69 5
22 16
96 76
30 96
9 13
22 39
63 5
29 48
91 74
84 17
85 94
94 96
82 92
88 51
83 24
28 51
66 52
79 2
97 13
77 61
99 6
10 51
98 13
87 38
7 40
33 22
69 43
34 27
29 50
25 21
24 64
67 53
54 41
54 45
5 50
7 82
32 56
68 16
6 81
20 6
79 12
6 5
83 48
49 39
11 11
26 74
26 86
79 92
89 31
32 32
65 93
100 39
6 97
77 20
25 11
74 49
53 56
3 79
69 35
97 51
76 32
69 23
66 1
83 47
30 54
99 15
53 95
43 98
17 10
21 19
19 94
1 46
77 71
57 3
5 97
28 6
80 81
33 97
43 45
51 16
82 25
19 49
98 86
95 61
31 67
