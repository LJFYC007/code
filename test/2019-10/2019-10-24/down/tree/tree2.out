12867801
17641666
2021891
1959525
17720862
8327357
19489613
15970874
176576
7333104
3884822
18732328
14153168
18449904
9776602
15106680
13481495
8748678
10651622
18089245
13784564
2311914
8357261
1864229
9406872
8653076
19490152
5224496
10510736
9808226
7461676
14537127
534143
16755981
16605180
10747494
13946604
1573857
1117292
2541207
199600
1236360
3503177
11601320
797
13516782
14887366
14345573
9155673
9980023
2527146
10548977
14383993
3340687
6772398
9377618
19408417
8877005
15719151
720
16206414
18266366
12500101
3106097
19490301
2533214
16004590
14867486
7987811
2021379
6180793
19490362
4535951
14164894
13764933
4706334
183608
4469011
8877441
1420686
15802672
17607818
2400851
11814350
19482886
6869687
7467014
17191822
10130746
13760584
14012224
842
8825780
18449904
13778831
18711544
2862954
10573827
469878
17735709
